README.md

```diff
@@ -229,7 +230,16 @@ A collection of android security related resources.
 3. [android app security checklist](https://github.com/b-mueller/android_app_security_checklist)
 4. [Mobile App Pentest Cheat Sheet](https://github.com/tanprathan/MobileApp-Pentest-Cheatsheet)
 5. [Android Reverse Engineering 101 by Daniele Altomare (Web Archive link)](http://web.archive.org/web/20180721134044/http://www.fasteque.com:80/android-reverse-engineering-101-part-1/)
-5. ~~[Mobile Security Reading Room](https://mobile-security.zeef.com) - A reading room which contains well categorised technical reading material about mobile penetration testing, mobile malware, mobile forensics and all kind of mobile security related topics~~
+6. [Android security checklist: WebView](https://blog.oversecured.com/Android-security-checklist-webview/)
```

Comment:
This repo is not meant as an advertisement for oversecured.com

Reply:
Hey @ashishb,

The previous item is an article but not a collection of articles. I initially wanted to add something like "[Oversecured blogs](https://blog.oversecured.com/)" to this section, but I considered this to be an advertisement.

How would you recommend noting the articles? They consider cool technical write-ups regarding Android security.

Thanks!